src/components/charts/other.js

```diff
@@ -248,11 +253,38 @@ export default class Other extends Component {
   }
 
   updateGaugeConfiguration() {
+    const { options, configurations } = this.state;
+
+    const { chars, digits, symbols } = analyzeGaugeText(configurations.gauge);
+
+    options.series[0].data[0].y = chars > 100 ? 100 : chars;
+    options.series[1].data[0].y = digits > 100 ? 100 : digits;
+    options.series[2].data[0].y = symbols > 100 ? 100 : symbols;
+
     this.setState({ rerenderChart: true }, () => {
       this.setState({ rerenderChart: false })
     })
   }
 
+  refreshGaugeCongifuration() {
+    const { options, configurations } = this.state;
+
+    configurations.gauge.text = 'Enter here your text to see its char analysis on the chart...';
+    configurations.gauge.chars = 500;
+    configurations.gauge.digits = 50;
+    configurations.gauge.symbols = 100;
+
+    const { chars, digits, symbols } = analyzeGaugeText(configurations.gauge);
+
+    options.series[0].data[0].y = chars > 100 ? 100 : chars;
+    options.series[1].data[0].y = digits > 100 ? 100 : digits;
+    options.series[2].data[0].y = symbols > 100 ? 100 : symbols;
```

Comment:
I do not really like that these lines are repeating lines from the previous method, but I did not get any fast and easy way how to avoid it... :(